scripts/download_incr_DB.py

```diff
@@ -140,7 +140,18 @@ def GetAllObjectsFromS3(url, folderName=""):
 def GetPersistenceKey(key_url):
 	retry_counter = 0
 	while True:
-		response = requests.get(key_url, stream=True)
+		try:
+		    response = requests.get(key_url, stream=True)
+		except Exception as e:
+			print(e)
+			print("Exception occurred while downloading file.Retrying: " + key_url)
```

Comment:
Because you have multiple threads running, it's possible this second `print` statement will not be shown immediately after the previous `print` statement. So it's better to combine them, like:

`print("Exception occurred while downloading " + key_url + ": " + str(e))`

Reply:
test log with new changes.
Exception occurred while downloading http://c5b68604-8540-4887-ad29-2ab9e680f997.s3.amazonaws.com/incremental/try640withubuntu18/persistence/txBlocks/MANIFEST-001503: Raising Exception